app/src/main/java/tr/edu/anadolu/bk/android/TabFragmentPagerAdapter.java

```diff
@@ -0,0 +1,35 @@
+package tr.edu.anadolu.bk.android;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+
+import tr.edu.anadolu.bk.android.fragment.PageFragment;
+import tr.edu.anadolu.bk.android.viewmodel.ViewPagerModel;
+
+public class TabFragmentPagerAdapter extends FragmentPagerAdapter {
+
+    private Context context;
+
+    public TabFragmentPagerAdapter(FragmentManager fm, Context context) {
+        super(fm);
+        this.context = context;
+    }
+
+    @Override
+    public int getCount() {
+        return ViewPagerModel.values().length;
+    }
+
+    @Override
+    public Fragment getItem(int position) {
```

Comment:
bu viewpager için fragmentList falan gondersek bence daha iyi oalcak. sonucta tek bir fragment kullanmaycagız.